client/extensions/woocommerce/app/settings/taxes/index.js

```diff
@@ -100,15 +100,18 @@ class SettingsTaxes extends Component {
 		return this.state.userBeganEditing;
 	}
 
-	onSave = ( event ) => {
+	onSave = ( event, onSuccessExtra ) => {
 		const { site, translate } = this.props;
 
 		event.preventDefault();
 		this.setState( { isSaving: true } );
 
 		const onSuccess = () => {
 			this.setState( { isSaving: false, userBeganEditing: false } );
-			return successNotice( translate( 'Settings updated successfully.' ), { duration: 4000 } );
+			if ( onSuccessExtra ) {
+				onSuccessExtra();
+			}
+			return successNotice( translate( 'Settings updated successfully.' ), { duration: 4000, displayOnNextPage: true } );
```

Comment:
Hi! I've found a possible matching string that has already been translated **24** times:
`translate( 'Theme settings updated successfully.' )` *ES Score: 14*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/woocommerce-settings-dashboard-button)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).